dotcom-rendering/makefile

```diff
@@ -32,45 +32,13 @@ build: clean-dist install
 	$(call log, "generating Islands report card")
 	@node ./scripts/islands/island-descriptions.mjs
 
-
-build-ci: clean-dist install
-	$(call log, "building production bundles")
-	@NODE_ENV=production webpack --config ./scripts/webpack/webpack.config.js
-
-start-ci: install
-	$(call log, "starting PROD server...")
-	@NODE_ENV=production DISABLE_LOGGING_AND_METRICS=true node dist/server.js
-
-start: install
-	$(call log, "starting PROD server...")
-	@echo '' # just a spacer
-	NODE_ENV=production pm2 start dist/server.js
-	@echo '' # just a spacer
-	$(call log, "PROD server is running")
-	@NODE_ENV=production pm2 logs
-
-# Used in cloudformation.yml which specifically:
-# - Launches pm2 from /usr/local/node rather than /node_modules
-start-prod:
+prod:
 	@echo 'starting PROD server...'
 	@echo '' # just a spacer
-	NODE_ENV=production /usr/local/node/pm2 start dist/server.js
-	@echo '' # just a spacer
-	@echo 'PROD server is running'
-
-stop:
-	@pm2 kill
-
-monitor:
-	@pm2 monit
-
-logs:
-	@pm2 logs
-
-run: stop build start
-
-run-ci: stop build-ci start-ci
+	NODE_ENV=production node dist/server.js
 
+ci: export DISABLE_LOGGING_AND_METRICS = true
+ci: prod
```

Comment:
I’ve not seen this double task name elsewhere, might be worth making it more similar to other tasks?

Reply:
So this is a [target specific variable](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html)

It's useful here as it applies the env variable to this specific target _before_ we run the  `prod` dependency.

I wanted to reuse the `prod` target for CI to have one single way of starting the app but with the addition of `DISABLE_LOGGING_AND_METRICS = true`.

Reply:
Tried that but it is invalid syntax:
`makefile:41: *** commands commence before first target.  Stop.`